CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Django CTE change log
 
+## v1.2.0 - 2022-03-30
+
+- Add support for Django 3.1, 3.2 and 4.0.
```

Comment:
Just to note, this says Django 4.0, but python tests for Django 4.0 weren't added. I assume that's intentional, but wanted to highlight in case not

Reply:
Tests for 4.0 were added in #43

Reply:
I see, my mistake!